Date/TimeZone.php

```diff
@@ -414,7 +413,7 @@ function Date_TimeZone($ps_id)
      * @return   object     Date_TimeZone object of the default time zone
      * @access   public
      */
-    function getDefault()
+    static function getDefault()
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.